PixiEditor/Models/Layers/Layer.cs

```diff
@@ -58,6 +59,16 @@ public Layer(string name, WriteableBitmap layerBitmap)
 
         public Dictionary<Coordinates, Color> LastRelativeCoordinates { get; set; }
 
+        public string LayerHighlightColor
+        {
+            get => IsActive ? layerHighlightColor : "#00000000";
+            set
+            {
+                layerHighlightColor = value;
+                RaisePropertyChanged(nameof(LayerHighlightColor));

```

Comment:
You could use the new SetProperty Method that I added in undo selection here

```suggestion
                SetProperty(ref layerHighlightColor, value);
```